openforcefield/__init__.py

```diff
@@ -1,4 +1,17 @@
+import warnings
+
 from ._version import get_versions
 
 __version__ = get_versions()["version"]
 del get_versions
+
+
+warnings.warn(
+    "Importing this package as `import openforcefield.XXX` and "
```

Comment:
Non-blocking, but it would be nice to have more precise language here
* The entire package is never really imported
* Saying "importing this module as ... " is incorrect when only a class is imported
* Likewise, "importing this class/etc. ..." is also incorrect when entire modules are imported

Maybe something more general like "Importing components from this package as ... " / "From version `0.9.0` onwards, components of this package will need to be imported as ..." ?